test.py

```diff
@@ -7,19 +7,27 @@
 import subprocess
 import tempfile
 from collections import defaultdict
-from enum import Flag
+from enum import IntFlag
 from functools import partial
 
 from elftools.elf.elffile import ELFFile
 from elftools.elf.sections import SymbolTableSection
 from tqdm.contrib.concurrent import process_map
 
 
-class KeepLogType(Flag):
```

Comment:
why the change to `IntFlag`?

Reply:
I though pt that it would be more convenient, as it also allows `x & KeepLogType.STDERR` if x is and int, but since we convert the arguments to `KeepLogType` anyways it has no real benefit.